shogun-data

```diff
@@ -0,0 +1 @@
+Subproject commit 9b12ffc6add78290c135025f913fab488657ff5d
```

Comment:
Ah, not 100%.
This hash points to a PR by somebody else that is more recent than yours.
Your hash was a different one. You can either revert the data repo to your hash and then update the subproject again, or manually put your hash in.

It doesnt really matter here as we will merge all the data and cookbooks anyways, but you need to make sure that you are in controll of this